docs/source/getting_started/depth_estimation/01_loading_depth_data.ipynb

```
@@ -0,0 +1,670 @@
+    "    \"\"\"\n",
+    "    if isinstance(src, list):\n",
+    "        return [os.path.join(prefix, item) for item in src]\n",
+    "    elif isinstance(src, dict):\n",
+    "        results = []\n",
+    "        for k, v in src.items():\n",
+    "            new_prefix = os.path.join(prefix, k) if prefix else k\n",
+    "            results.extend(enumerate_paths(v, new_prefix))\n",
+    "        return results\n",
+    "    else:\n",
+    "        raise ValueError(f'Unexpected data type: {type(src)}')"
+   ]
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)